src/generated/resources/data/forge/tags/blocks/candles.json

```diff
@@ -0,0 +1,22 @@
+{
+  "replace": false,
+  "values": [
+    "minecraft:candle",
+    "minecraft:black_candle",
+    "minecraft:blue_candle",
+    "minecraft:brown_candle",
+    "minecraft:cyan_candle",
+    "minecraft:gray_candle",
+    "minecraft:green_candle",
+    "minecraft:light_blue_candle",
+    "minecraft:light_gray_candle",
+    "minecraft:lime_candle",
+    "minecraft:magenta_candle",
+    "minecraft:orange_candle",
+    "minecraft:pink_candle",
+    "minecraft:purple_candle",
+    "minecraft:red_candle",
+    "minecraft:white_candle",
+    "minecraft:yellow_candle"
+  ]
+}
```

Comment:
Why does forge need to add it's own item & block tags for candles when the tag `minecraft:candles` already exists? Shouldn't mods that add their own candles simply add to the vanilla tag like they do with `minecraft:logs`?

Reply:
I didn't see `minecraft:candles`, I must have overlooked it.